.travis.yml

```diff
@@ -2,7 +2,6 @@ language: node_js
 sudo: false
 node_js:
   - 4
-  - 6
```

Comment:
node-sass does not work on node 6. I added [here](https://github.com/bigcommerce/stencil-styles/pull/21) by mistake